x-pack/plugins/security_solution/common/ecs/process/index.ts

```diff
@@ -4,7 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { Ext } from '../file';
+
 export interface ProcessEcs {
+  Ext?: Ext;
```

Comment:
Same here - should we follow the standard of making it all lowercase here?

Reply:
Both this and the `Ransomware` fields are just following the schema based on the data coming back from the endpoint. It's populating [these fields](https://github.com/elastic/endpoint-package/blob/master/custom_schemas/custom_ransomware.yml#L9) so this is just mirroring those changes